### ISSUE TITLE
[Bug] Handbrake keeps restarting

### ISSUE THREAD

**Issue by TNFB** [jlesage/docker-handbrake#362]
### Current Behavior

Hello,
I was doing some automatic conversions when suddenly Handbrake crashed and refused to start without any action on my part. It then went into an infinite restart loop.
I tried several things but none of them worked (like re-pulling the image and so on).

### Environment

- OS: Debian 12 Bookworm
- OS version: kernel 6.7.12-1~bpo12+1
- CPU: AMD Ryzen 5 2600X
- Docker version: 26.1.3, build b72abbb
- GPU : Intel Arc A380
- Device model: N.A
- Browser/OS: N.A

### Container creation

  handbrake:
    container_name: handbrake
    image: jlesage/handbrake
    restart: unless-stopped
    depends_on:
      - wireguard
    network_mode: service:wireguard
    environment:
      - TZ=Europe/Paris
      - PUID=1000
      - PGID=1000
      - VNC_LISTENING_PORT=-1
      - SECURE_CONNECTION=0
      - HANDBRAKE_DEBUG=1
      - AUTOMATED_CONVERSION=0
      - AUTOMATED_CONVERSION_PRESET=transcode/av1qsv10bit
      - AUTOMATED_CONVERSION_OUTPUT_SUBDIR=SAME_AS_SRC
      - AUTOMATED_CONVERSION_FORMAT=mkv
      - CONTAINER_DEBUG=1
      - DARK_MODE=1
      - LANG=fr_FR.UTF-8
      - UMASK=0002
    volumes:
      - /handbrake/config:/config:rw
      - /defaultpath:/storage:ro
      - /pathwatching:/watch:r
      - /pathoutput:/output:rw
    devices:
      - /dev/dri:/dev/dri

### Container log

```text
[init        ] container is starting...
[cont-env    ] loading container environment variables...
[cont-env    ] APP_NAME: loading...
[cont-env    ] APP_VERSION: loading...
[cont-env    ] DISPLAY: executing...
[cont-env    ] DISPLAY: terminated successfully.
[cont-env    ] DISPLAY: loading...
[cont-env    ] DOCKER_IMAGE_PLATFORM: loading...
[cont-env    ] DOCKER_IMAGE_VERSION: loading...
[cont-env    ] GTK2_RC_FILES: executing...
[cont-env    ] GTK2_RC_FILES: terminated successfully.
[cont-env    ] GTK2_RC_FILES: loading...
[cont-env    ] GTK_THEME: executing...
[cont-env    ] GTK_THEME: terminated successfully.
[cont-env    ] GTK_THEME: loading...
[cont-env    ] HOME: loading...
[cont-env    ] INSTALL_PACKAGES_INTERNAL: executing...
[cont-env    ] INSTALL_PACKAGES_INTERNAL: terminated successfully.
[cont-env    ] INSTALL_PACKAGES_INTERNAL: loading...
[cont-env    ] QT_STYLE_OVERRIDE: executing...
[cont-env    ] QT_STYLE_OVERRIDE: terminated successfully.
[cont-env    ] QT_STYLE_OVERRIDE: loading...
[cont-env    ] SUP_GROUP_IDS_INTERNAL: executing...
[cont-env    ] SUP_GROUP_IDS_INTERNAL: terminated successfully.
[cont-env    ] SUP_GROUP_IDS_INTERNAL: loading...
[cont-env    ] TAKE_CONFIG_OWNERSHIP: loading...
[cont-env    ] XDG_CACHE_HOME: loading...
[cont-env    ] XDG_CONFIG_HOME: loading...
[cont-env    ] XDG_DATA_HOME: loading...
[cont-env    ] XDG_RUNTIME_DIR: loading...
[cont-env    ] XDG_STATE_HOME: loading...
[cont-env    ] container environment variables initialized.
[cont-secrets] loading container secrets...
[cont-secrets] container secrets loaded.
[cont-init   ] executing container initialization scripts...
[cont-init   ] 10-certs.sh: executing...
[cont-init   ] 10-certs.sh: terminated successfully.
[cont-init   ] 10-check-app-niceness.sh: executing...
[cont-init   ] 10-check-app-niceness.sh: terminated successfully.
[cont-init   ] 10-clean-logmonitor-states.sh: executing...
[cont-init   ] 10-clean-logmonitor-states.sh: terminated successfully.
[cont-init   ] 10-clean-tmp-dir.sh: executing...
[cont-init   ] 10-clean-tmp-dir.sh: terminated successfully.
[cont-init   ] 10-fontconfig-cache-dir.sh: executing...
[cont-init   ] 10-fontconfig-cache-dir.sh: terminated successfully.
[cont-init   ] 10-init-users.sh: executing...
[cont-init   ] 10-init-users.sh: terminated successfully.
[cont-init   ] 10-nginx.sh: executing...
[cont-init   ] 10-nginx.sh: terminated successfully.
[cont-init   ] 10-openbox.sh: executing...
[cont-init   ] 10-openbox.sh: terminated successfully.
[cont-init   ] 10-pkgs-mirror.sh: executing...
[cont-init   ] 10-pkgs-mirror.sh: terminated successfully.
[cont-init   ] 10-set-tmp-dir-perms.sh: executing...
[cont-init   ] 10-set-tmp-dir-perms.sh: terminated successfully.
[cont-init   ] 10-vnc-password.sh: executing...
[cont-init   ] 10-vnc-password.sh: terminated successfully.
[cont-init   ] 10-web-data.sh: executing...
[cont-init   ] 10-web-data.sh: terminated successfully.
[cont-init   ] 10-x11-unix.sh: executing...
[cont-init   ] 10-x11-unix.sh: terminated successfully.
[cont-init   ] 10-xdg-runtime-dir.sh: executing...
[cont-init   ] 10-xdg-runtime-dir.sh: terminated successfully.
[cont-init   ] 15-cjk-font.sh: executing...
[cont-init   ] 15-cjk-font.sh: terminated successfully.
[cont-init   ] 15-install-pkgs.sh: executing...
[cont-init   ] 15-install-pkgs.sh: terminated successfully.
[cont-init   ] 54-check-optical-drive.sh: executing...
[cont-init   ] 54-check-optical-drive.sh: looking for usable optical drives...
[cont-init   ] 54-check-optical-drive.sh: no usable optical drive found.
[cont-init   ] 54-check-optical-drive.sh: terminated successfully.
[cont-init   ] 54-check-qsv.sh: executing...
[cont-init   ] 54-check-qsv.sh: Processor: AMD Ryzen 5 2600X Six-Core Processor
[cont-init   ] 54-check-qsv.sh: Microarchitecture: AMD_ZEN_PLUS
[cont-init   ] 54-check-qsv.sh: Kernel: 6.7.12+bpo-amd64
[cont-init   ] 54-check-qsv.sh: Intel Quick Sync Video may not be supported: processor not QSV capable.
[cont-init   ] 54-check-qsv.sh: terminated successfully.
[cont-init   ] 54-check-trash-dir.sh: executing...
[cont-init   ] 54-check-trash-dir.sh: terminated successfully.
[cont-init   ] 55-handbrake.sh: executing...
[cont-init   ] 55-handbrake.sh: core dump file location: core
[cont-init   ] 55-handbrake.sh: core dump file size: unlimited (blocks)
[cont-init   ] 55-handbrake.sh: terminated successfully.
[cont-init   ] 85-take-config-ownership.sh: executing...
[cont-init   ] 85-take-config-ownership.sh: terminated successfully.
[cont-init   ] 89-info.sh: executing...
    ╭――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――╮
    │                                                                      │
    │ Application:           HandBrake                                     │
    │ Application Version:   1.7.3                                         │
    │ Docker Image Version:  24.03.1                                       │
    │ Docker Image Platform: linux/amd64                                   │
    │                                                                      │
    ╰――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――╯
[cont-init   ] 89-info.sh: terminated successfully.
[cont-init   ] all container initialization scripts executed.
[init        ] giving control to process supervisor.
[supervisor  ] loading services...
[supervisor  ] loading service 'default'...
[supervisor  ] loading service 'logmonitor'...
[supervisor  ] service 'logmonitor' is disabled.
[supervisor  ] loading service 'app'...
[supervisor  ] loading service 'gui'...
[supervisor  ] loading service 'certsmonitor'...
[supervisor  ] service 'certsmonitor' is disabled.
[supervisor  ] loading service 'xvnc'...
[supervisor  ] loading service 'nginx'...
[supervisor  ] loading service 'openbox'...
[supervisor  ] loading service 'logrotate'...
[supervisor  ] loading service 'autovideoconverter'...
[supervisor  ] service 'autovideoconverter' is disabled.
[supervisor  ] all services loaded.
[supervisor  ] starting services...
[supervisor  ] starting service 'xvnc'...
[supervisor  ] started service 'xvnc'.
[xvnc        ] Sun Jun 23 23:43:44 2024
[xvnc        ]  Config:      set rfbport(Int) to -1
[xvnc        ]  Config:      set rfbunixpath(String) to /tmp/vnc.sock
[xvnc        ]  Config:      set rfbunixmode(Int) to 0660
[xvnc        ]  Config:      set SecurityTypes(String) to None
[xvnc        ]  Config:      set desktop(String) to HandBrake
[xvnc        ] Xvnc TigerVNC 1.13.1 - built Dec 21 2023 00:54:05
[xvnc        ] Copyright (C) 1999-2022 TigerVNC Team and many others (see README.rst)
[xvnc        ] See https://www.tigervnc.org for information on TigerVNC.
[xvnc        ] Underlying X server release 12014000
[xvnc        ]  vncext:      VNC extension running!
[xvnc        ]  Config:      set immutable AllowOverride
[xvnc        ]  vncext:      Listening for VNC connections on /tmp/vnc.sock (mode 0660)
[xvnc        ]  VNCServerST: creating single-threaded server HandBrake
[xvnc        ]  vncext:      created VNC server for screen 0
[supervisor  ] waiting for service 'xvnc' to be ready...
[supervisor  ] starting service 'nginx'...
[supervisor  ] started service 'nginx'.
[nginx       ] Listening for HTTP connections on port 5800.
[supervisor  ] waiting for service 'nginx' to be ready...
[supervisor  ] starting service 'openbox'...
[supervisor  ] started service 'openbox'.
[xvnc        ] Sun Jun 23 23:43:45 2024
[xvnc        ]  Selection:   Selection owner change for WM_S0
[supervisor  ] waiting for service 'openbox' to be ready...
[supervisor  ] starting service 'app'...
[supervisor  ] started service 'app'.
[supervisor  ] starting service 'logrotate'...
[supervisor  ] started service 'logrotate'.
[supervisor  ] all services started.
[logrotate   ] reading config file /opt/base/etc/logrotate.conf
[logrotate   ] including /etc/cont-logrotate.d
[logrotate   ] reading config file nginx
[logrotate   ] acquired lock on state file /config/xdg/state/logrotate/logrotate.status
[logrotate   ] Reading state from file: /config/xdg/state/logrotate/logrotate.status
[logrotate   ] Allocating hash table for state file, size 64 entries
[logrotate   ] Creating new state
[logrotate   ] Creating new state
[logrotate   ] Handling 1 logs
[logrotate   ] rotating pattern: /config/log/nginx/access.log /config/log/nginx/error.log  monthly (6 rotations)
[logrotate   ] empty log files are not rotated, old logs are removed
[logrotate   ] considering log /config/log/nginx/access.log
[logrotate   ]   Now: 2024-06-23 23:43
[logrotate   ]   Last rotated at 2024-06-04 12:00
[logrotate   ]   log does not need rotating (log has been rotated at 2024-06-04 12:00, which is less than a month ago)
[logrotate   ] considering log /config/log/nginx/error.log
[logrotate   ]   Now: 2024-06-23 23:43
[logrotate   ]   Last rotated at 2024-06-04 12:00
[logrotate   ]   log does not need rotating (log has been rotated at 2024-06-04 12:00, which is less than a month ago)
[logrotate   ] not running postrotate script, since no logs were rotated
[supervisor  ] service 'app' exited (got signal SIGSEGV).
[supervisor  ] waiting termination of logger thread of service 'app'...
[supervisor  ] logger thread of service 'app' successfully terminated.
[supervisor  ] service 'app' exited, shutting down...
[supervisor  ] service 'logrotate' exited (with status 0).
[supervisor  ] waiting termination of logger thread of service 'logrotate'...
[supervisor  ] logger thread of service 'logrotate' successfully terminated.
[supervisor  ] stopping service 'openbox'...
[supervisor  ] service 'openbox' exited (with status 0).
[supervisor  ] waiting termination of logger thread of service 'openbox'...
[supervisor  ] logger thread of service 'openbox' successfully terminated.
[supervisor  ] stopping service 'nginx'...
[supervisor  ] service 'nginx' exited (with status 0).
[supervisor  ] waiting termination of logger thread of service 'nginx'...
[supervisor  ] logger thread of service 'nginx' successfully terminated.
[supervisor  ] stopping service 'xvnc'...
[xvnc        ] Sun Jun 23 23:43:47 2024
[xvnc        ]  VNCServerST: shutting down server HandBrake
[xvnc        ]  ComparingUpdateTracker: 0 pixels in / 0 pixels out
[xvnc        ]  ComparingUpdateTracker: (1:-nan ratio)
[supervisor  ] service 'xvnc' exited (with status 0).
[supervisor  ] waiting termination of logger thread of service 'xvnc'...
[supervisor  ] logger thread of service 'xvnc' successfully terminated.
[finish      ] executing container finish scripts...
[finish      ] all container finish scripts executed.
```

### Container inspect

```text
[
    {
        "Id": "fee82a0520a2f063378fe7cbe33cd1e94b5a591840f3bed16d5e796103f45a87",
        "Created": "2024-06-23T21:33:51.62860903Z",
        "Path": "/init",
        "Args": [],
        "State": {
            "Status": "restarting",
            "Running": true,
            "Paused": false,
            "Restarting": true,
            "OOMKilled": false,
            "Dead": false,
            "Pid": 0,
            "ExitCode": 139,
            "Error": "",
            "StartedAt": "2024-06-23T21:44:48.178870094Z",
            "FinishedAt": "2024-06-23T21:44:52.213911006Z"
        },
        "Image": "sha256:999a8e3a055e59eb7fe52bbd69691c10204bd65bc9592492d64ff1b0984a3c84",
        "ResolvConfPath": "/Disques/SSD2/docker/docker/containers/d3b205069b4de5cd7f0607f521855f41b8ea1ecdd03a2523bd39a9a87d6de0a2/resolv.conf",
        "HostnamePath": "/Disques/SSD2/docker/docker/containers/d3b205069b4de5cd7f0607f521855f41b8ea1ecdd03a2523bd39a9a87d6de0a2/hostname",
        "HostsPath": "/Disques/SSD2/docker/docker/containers/d3b205069b4de5cd7f0607f521855f41b8ea1ecdd03a2523bd39a9a87d6de0a2/hosts",
        "LogPath": "/Disques/SSD2/docker/docker/containers/fee82a0520a2f063378fe7cbe33cd1e94b5a591840f3bed16d5e796103f45a87/fee82a0520a2f063378fe7cbe33cd1e94b5a591840f3bed16d5e796103f45a87-json.log",
        "Name": "/handbrake",
        "RestartCount": 19,
        "Driver": "overlay2",
        "Platform": "linux",
        "MountLabel": "",
        "ProcessLabel": "",
        "AppArmorProfile": "docker-default",
        "ExecIDs": null,
        "HostConfig": {
            "Binds": [
                "/pathwatch:/watch:rw",
                "/pathoutput:/output:rw",
                "/handbrake/config:/config:rw",
                "/defaultpath:/storage:ro"
            ],
            "ContainerIDFile": "",
            "LogConfig": {
                "Type": "json-file",
                "Config": {}
            },
            "NetworkMode": "container:d3b205069b4de5cd7f0607f521855f41b8ea1ecdd03a2523bd39a9a87d6de0a2",
            "PortBindings": {},
            "RestartPolicy": {
                "Name": "unless-stopped",
                "MaximumRetryCount": 0
            },
            "AutoRemove": false,
            "VolumeDriver": "",
            "VolumesFrom": null,
            "ConsoleSize": [
                0,
                0
            ],
            "CapAdd": null,
            "CapDrop": null,
            "CgroupnsMode": "private",
            "Dns": null,
            "DnsOptions": null,
            "DnsSearch": null,
            "ExtraHosts": [],
            "GroupAdd": null,
            "IpcMode": "private",
            "Cgroup": "",
            "Links": null,
            "OomScoreAdj": 0,
            "PidMode": "",
            "Privileged": false,
            "PublishAllPorts": false,
            "ReadonlyRootfs": false,
            "SecurityOpt": null,
            "UTSMode": "",
            "UsernsMode": "",
            "ShmSize": 67108864,
            "Runtime": "runc",
            "Isolation": "",
            "CpuShares": 0,
            "Memory": 0,
            "NanoCpus": 0,
            "CgroupParent": "",
            "BlkioWeight": 0,
            "BlkioWeightDevice": null,
            "BlkioDeviceReadBps": null,
            "BlkioDeviceWriteBps": null,
            "BlkioDeviceReadIOps": null,
            "BlkioDeviceWriteIOps": null,
            "CpuPeriod": 0,
            "CpuQuota": 0,
            "CpuRealtimePeriod": 0,
            "CpuRealtimeRuntime": 0,
            "CpusetCpus": "",
            "CpusetMems": "",
            "Devices": [
                {
                    "PathOnHost": "/dev/dri",
                    "PathInContainer": "/dev/dri",
                    "CgroupPermissions": "rwm"
                }
            ],
            "DeviceCgroupRules": null,
            "DeviceRequests": null,
            "MemoryReservation": 0,
            "MemorySwap": 0,
            "MemorySwappiness": null,
            "OomKillDisable": null,
            "PidsLimit": null,
            "Ulimits": null,
            "CpuCount": 0,
            "CpuPercent": 0,
            "IOMaximumIOps": 0,
            "IOMaximumBandwidth": 0,
            "MaskedPaths": [
                "/proc/asound",
                "/proc/acpi",
                "/proc/kcore",
                "/proc/keys",
                "/proc/latency_stats",
                "/proc/timer_list",
                "/proc/timer_stats",
                "/proc/sched_debug",
                "/proc/scsi",
                "/sys/firmware",
                "/sys/devices/virtual/powercap"
            ],
            "ReadonlyPaths": [
                "/proc/bus",
                "/proc/fs",
                "/proc/irq",
                "/proc/sys",
                "/proc/sysrq-trigger"
            ]
        },
        "GraphDriver": {
            "Data": {
                "LowerDir": "/Disques/SSD2/docker/docker/overlay2/a97bd109c1c751d7345d18fa4706a734e207291983e6daa87131caf96234ca66-init/diff:/Disques/SSD2/docker/docker/overlay2/814146cc35b63d31f1410d2362835030ff92ceaaaed2bb2c7687d17e9a7f49fb/diff:/Disques/SSD2/docker/docker/overlay2/10788dcd9256aad2b93064da33233f2cf5dac760524063e60f9950286e9066fc/diff:/Disques/SSD2/docker/docker/overlay2/c3bd253015f1d46aeb14dca4cfe6d0f7ed079f4534a0f2065b1199b90c39b7da/diff:/Disques/SSD2/docker/docker/overlay2/86bc98b43c2279c429c7eee13c01fca0e379f6b53bdd1dbf0adb0de4935872b5/diff:/Disques/SSD2/docker/docker/overlay2/911832a8390792a3a75e0506c985a3f5529caeb7877d7a533f230eb620420d1e/diff:/Disques/SSD2/docker/docker/overlay2/ff79c01f4d47ec23c43fb839f38f0d34dac39b7e3cbd71a51b8fcbfa8315a4e5/diff:/Disques/SSD2/docker/docker/overlay2/4988c5eba3ab27280b8bc74e5f85f520725eb23d730912f93746cebb0e8a826c/diff:/Disques/SSD2/docker/docker/overlay2/1c178b2fdf3329f2155dfefca67c81505f98cbbfdad98a9640d53c859c6317f0/diff:/Disques/SSD2/docker/docker/overlay2/27b7e9fc3a215c76857aed3bb73f906c5a072e770070ad2706bcf4cdc5f0494d/diff:/Disques/SSD2/docker/docker/overlay2/0cf5c0ecaab634ea6581256c1869d17d66c6c38ececa9bc189f5c1e1240dae05/diff:/Disques/SSD2/docker/docker/overlay2/13a423399417221f87f059d515137c24e05a744675df1f7c220b31f2d9caff5b/diff:/Disques/SSD2/docker/docker/overlay2/d5d91571f85820ed137af8447162cd98174fca77304d661766d3b3352158f609/diff:/Disques/SSD2/docker/docker/overlay2/0d07e79bbd9884b093bb7398e3771f1b26fee16031a05b276b62782029cb3f64/diff:/Disques/SSD2/docker/docker/overlay2/6c2a3d6a2ece2acd68335a2c5e1c7f7bc25dcee82e5694993c995aefe5fe87c8/diff:/Disques/SSD2/docker/docker/overlay2/4208e56c6d46cad61fb34bc05762eb8a3fdb5e96cb9a01e8848f1684f07d83d1/diff:/Disques/SSD2/docker/docker/overlay2/47e10483ff58ac9a5d0797c4f072ffc10130039d2c71e5c31ce05cf6cc58e5ab/diff:/Disques/SSD2/docker/docker/overlay2/96a717cfde7f9bf56ddf8cc7aaf337eeb22a53f3c32d0c5f68f0d3b74c8e2ecc/diff:/Disques/SSD2/docker/docker/overlay2/f69aad35f6cde343b04b000583b60090595b3cb29700aed557aee73302c5957d/diff:/Disques/SSD2/docker/docker/overlay2/30a25c72fe8e1298a87601a5ea534de3151945b9ff3e1e2634cd60eae6e8f28d/diff:/Disques/SSD2/docker/docker/overlay2/e59fcdad13cabed74062276dd2d6c08f5bb7ee9ff73c1f9852cf79a2b371819b/diff:/Disques/SSD2/docker/docker/overlay2/70caaafdaa27f0fcb285c070ff9570fb10b4f88efd7e783a443536eece001f88/diff:/Disques/SSD2/docker/docker/overlay2/415f693df0153df0a8f03d0a2082e4b6db7d9eb93da2d91c7a01b3a99ff9f56f/diff:/Disques/SSD2/docker/docker/overlay2/11a721f3de178e27798452e6737a239e81f26cb69278ae7a147bfb54cf6a776c/diff:/Disques/SSD2/docker/docker/overlay2/974e7ca3c2388be10e29f037ec64815de45d02ad7502990706bea36f22a16231/diff:/Disques/SSD2/docker/docker/overlay2/3edba783a5770ef2ca9bd86e58b4c34fe0128e88cc34b7f39067c393943f5e2a/diff:/Disques/SSD2/docker/docker/overlay2/1fb4d7f33cf37af5145772f1d42cffcb6ddd9efb0df323eb111ffa6ae05ad035/diff:/Disques/SSD2/docker/docker/overlay2/463d34cabf5ba911fb43d95abad41f7538034dbefaca53501bdf5232d217dde1/diff:/Disques/SSD2/docker/docker/overlay2/9cb22fe9a37998df66885d881edadedeecb3aa7df627a8fa64c9bd89ed776ce9/diff:/Disques/SSD2/docker/docker/overlay2/3eeb50461befcc105d9e8c89cb1f5abd1214156b1c01cedd94d0e6e6e55cb61b/diff:/Disques/SSD2/docker/docker/overlay2/0845a78ac8423deb842a0f50bfcbc2078b04948904c60790c11f52c8e4ac5717/diff:/Disques/SSD2/docker/docker/overlay2/2b07ab78f12c6fe646149acd0e2a0c72519f1173b2b37f2f27ccbcfc3c4c954b/diff:/Disques/SSD2/docker/docker/overlay2/36f78101ff9d5efaca573f9e20696df58025cd7ebc88acc91bfa471af84743a0/diff:/Disques/SSD2/docker/docker/overlay2/cff5848ec8ba55a70e730c9e5af844a4cc22b265056b90a60af9ea73377efe4e/diff:/Disques/SSD2/docker/docker/overlay2/8a409d8ea90610f2b267a7f1a4bb8fd91b775bdf1db35d9c4307885b238f2d2a/diff:/Disques/SSD2/docker/docker/overlay2/cc866afe821f25ae8dcda234c2e8f8e5ddcb438d8e19ccdab28b227c65236efc/diff:/Disques/SSD2/docker/docker/overlay2/160e5b87f9f24ea7d04e7c46cfa77823f12a02575153452a88e8b63f000a9784/diff",
                "MergedDir": "/Disques/SSD2/docker/docker/overlay2/a97bd109c1c751d7345d18fa4706a734e207291983e6daa87131caf96234ca66/merged",
                "UpperDir": "/Disques/SSD2/docker/docker/overlay2/a97bd109c1c751d7345d18fa4706a734e207291983e6daa87131caf96234ca66/diff",
                "WorkDir": "/Disques/SSD2/docker/docker/overlay2/a97bd109c1c751d7345d18fa4706a734e207291983e6daa87131caf96234ca66/work"
            },
            "Name": "overlay2"
        },
        "Mounts": [
            {
                "Type": "bind",
                "Source": "/pathwatch",
                "Destination": "/watch",
                "Mode": "rw",
                "RW": true,
                "Propagation": "rprivate"
            },
            {
                "Type": "bind",
                "Source": "/pathoutput",
                "Destination": "/output",
                "Mode": "rw",
                "RW": true,
                "Propagation": "rprivate"
            },
            {
                "Type": "bind",
                "Source": "/handbrake/config",
                "Destination": "/config",
                "Mode": "rw",
                "RW": true,
                "Propagation": "rprivate"
            },
            {
                "Type": "bind",
                "Source": "/defaultpath",
                "Destination": "/storage",
                "Mode": "ro",
                "RW": false,
                "Propagation": "rprivate"
            },
            {
                "Type": "volume",
                "Name": "e56edc4b4cc3840fa260ee9c67c9a2b1d654a65afb028394cfd7a2bf6a7fd1b7",
                "Source": "/Disques/SSD2/docker/docker/volumes/e56edc4b4cc3840fa260ee9c67c9a2b1d654a65afb028394cfd7a2bf6a7fd1b7/_data",
                "Destination": "/trash",
                "Driver": "local",
                "Mode": "",
                "RW": true,
                "Propagation": ""
            }
        ],
        "Config": {
            "Hostname": "d3b205069b4d",
            "Domainname": "",
            "User": "",
            "AttachStdin": false,
            "AttachStdout": true,
            "AttachStderr": true,
            "ExposedPorts": {
                "5800/tcp": {},
                "5900/tcp": {}
            },
            "Tty": false,
            "OpenStdin": false,
            "StdinOnce": false,
            "Env": [
                "AUTOMATED_CONVERSION_OUTPUT_SUBDIR=SAME_AS_SRC",
                "AUTOMATED_CONVERSION=0",
                "DARK_MODE=1",
                "AUTOMATED_CONVERSION_FORMAT=mkv",
                "CONTAINER_DEBUG=1",
                "TZ=Europe/Paris",
                "LANG=fr_FR.UTF-8",
                "PGID=1000",
                "PUID=1000",
                "AUTOMATED_CONVERSION_PRESET=transcode/av1qsv10bit",
                "VNC_LISTENING_PORT=-1",
                "UMASK=0002",
                "SECURE_CONNECTION=0",
                "HANDBRAKE_DEBUG=1",
                "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/base/sbin:/opt/base/bin",
                "ENV=/root/.docker_rc",
                "USER_ID=1000",
                "GROUP_ID=1000",
                "SUP_GROUP_IDS=",
                "KEEP_APP_RUNNING=0",
                "APP_NICENESS=0",
                "INSTALL_PACKAGES=",
                "PACKAGES_MIRROR=",
                "DISPLAY_WIDTH=1920",
                "DISPLAY_HEIGHT=1080",
                "SECURE_CONNECTION_VNC_METHOD=SSL",
                "SECURE_CONNECTION_CERTS_CHECK_INTERVAL=60",
                "WEB_LISTENING_PORT=5800",
                "VNC_PASSWORD=",
                "ENABLE_CJK_FONT=0",
                "HANDBRAKE_GUI=1",
                "AUTOMATED_CONVERSION_SOURCE_STABLE_TIME=5",
                "AUTOMATED_CONVERSION_SOURCE_MIN_DURATION=10",
                "AUTOMATED_CONVERSION_SOURCE_MAIN_TITLE_DETECTION=0",
                "AUTOMATED_CONVERSION_KEEP_SOURCE=1",
                "AUTOMATED_CONVERSION_OUTPUT_DIR=/output",
                "AUTOMATED_CONVERSION_OVERWRITE_OUTPUT=0",
                "AUTOMATED_CONVERSION_VIDEO_FILE_EXTENSIONS=",
                "AUTOMATED_CONVERSION_NON_VIDEO_FILE_ACTION=ignore",
                "AUTOMATED_CONVERSION_NON_VIDEO_FILE_EXTENSIONS=jpg jpeg bmp png gif txt nfo",
                "AUTOMATED_CONVERSION_MAX_WATCH_FOLDERS=5",
                "AUTOMATED_CONVERSION_CHECK_INTERVAL=5",
                "AUTOMATED_CONVERSION_HANDBRAKE_CUSTOM_ARGS=",
                "AUTOMATED_CONVERSION_INSTALL_PKGS=",
                "AUTOMATED_CONVERSION_NO_GUI_PROGRESS=0",
                "AUTOMATED_CONVERSION_USE_TRASH=0"
            ],
            "Cmd": [
                "/init"
            ],
            "Image": "jlesage/handbrake",
            "Volumes": {
                "/config": {},
                "/output": {},
                "/storage": {},
                "/trash": {},
                "/watch": {}
            },
            "WorkingDir": "/tmp",
            "Entrypoint": null,
            "OnBuild": null,
            "Labels": {
                "com.docker.compose.config-hash": "f56a1c243d1b26501e9a5ada22160acea5ea1c60bb58845844b11b9c8bc5728a",
                "com.docker.compose.container-number": "1",
                "com.docker.compose.depends_on": "wireguard:service_started:false",
                "com.docker.compose.image": "sha256:999a8e3a055e59eb7fe52bbd69691c10204bd65bc9592492d64ff1b0984a3c84",
                "com.docker.compose.oneoff": "False",
                "com.docker.compose.project": "tomflix",
                "com.docker.compose.project.config_files": "/home/chimorin/docker-compose/vpn-transmission-docker-compose.yml",
                "com.docker.compose.project.working_dir": "/home/chimorin/docker-compose",
                "com.docker.compose.service": "handbrake",
                "com.docker.compose.version": "2.27.0",
                "org.label-schema.description": "Docker container for HandBrake",
                "org.label-schema.name": "handbrake",
                "org.label-schema.schema-version": "1.0",
                "org.label-schema.vcs-url": "https://github.com/jlesage/docker-handbrake",
                "org.label-schema.version": "24.03.1"
            }
        },
        "NetworkSettings": {
            "Bridge": "",
            "SandboxID": "",
            "SandboxKey": "",
            "Ports": {},
            "HairpinMode": false,
            "LinkLocalIPv6Address": "",
            "LinkLocalIPv6PrefixLen": 0,
            "SecondaryIPAddresses": null,
            "SecondaryIPv6Addresses": null,
            "EndpointID": "",
            "Gateway": "",
            "GlobalIPv6Address": "",
            "GlobalIPv6PrefixLen": 0,
            "IPAddress": "",
            "IPPrefixLen": 0,
            "IPv6Gateway": "",
            "MacAddress": "",
            "Networks": {}
        }
    }
]
```

### Anything else?

In my docker compose, I have these lines :
    depends_on:
      - wireguard
    network_mode: service:wireguard
 It's because I run my platform behind a VPN and its the VPN who redirect port. This configuration was running correctly before.

Furthermore, here's what's in the Handbrake debug file :

(fr.handbrake.ghb:847): ghb-DEBUG: 23:22:30.968: Function: create_builder_or_die
(fr.handbrake.ghb:847): GLib-GIO-DEBUG: 23:22:31.112: _g_io_module_get_default: Found default implementation keyfile (GKeyfileSettingsBackend) for ‘gsettings-backend’
(fr.handbrake.ghb:847): GLib-GIO-DEBUG: 23:22:31.216: Failed to initialize portal (GPowerProfileMonitorPortal) for gio-power-profile-monitor: Not using portals
(fr.handbrake.ghb:847): GLib-GIO-DEBUG: 23:22:31.216: _g_io_module_get_default: Found default implementation dbus (GPowerProfileMonitorDBus) for ‘gio-power-profile-monitor’
(fr.handbrake.ghb:847): ghb-DEBUG: 23:22:31.216: Function: ghb_ui_update (activity_location)
(fr.handbrake.ghb:847): ghb-DEBUG: 23:22:31.216: Function: ghb_get_setting_key
(fr.handbrake.ghb:847): ghb-DEBUG: 23:22:31.216: Function: ghb_widget_to_setting
(fr.handbrake.ghb:847): ghb-DEBUG: 23:22:31.216: Function: ghb_get_setting_key
(fr.handbrake.ghb:847): ghb-DEBUG: 23:22:31.216: Function: ghb_get_setting_key

**Comment by jlesage** [comment]
Is it working if you start with a new `/config` ?

**Comment by TNFB** [comment]
Hey !
I experimented on my side and even by completely changing the location of the /config (I deleted everything and let Handbrake recreate a default configuration), nothing worked...
However, I found an (unfortunate) stupid solution to my problem: reboot the computer...
After that, everything started working properly again, or almost. Now, I have files that come out in "Conversion ended successfully" but with a size of 0kb. I don't think it's really related to my post though. I've seen this problem in a few other topics but I don't really know if it's a bug and if there's a fix.
Anyway, thanks for replying!
